Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -5430,7 +5472,15 @@ def panorama_check_latest_panos_software_command(target: Optional[str] = None):
         'GET',
         params=params
     )
-    return_results(result['response']['result'])
+    to_context = result.get('response', {}).get('result', {})
+    versions = to_context.get('sw-updates', {}).get('versions').get('entry', [])
+    if len(versions) > 5:
+        versions = versions[:5]
+    human_readable = tableToMarkdown('5 latest pan-os software releases', versions, ['version', 'filename', 'size', 'released-on', 'downloaded' , 'current' , 'latest', 'uploaded'] )
+    return CommandResults(readable_output=human_readable,
+                                  outputs=to_context,
+                                  raw_response=result,
+                                  )
```

Comment:
```suggestion
    return CommandResults(readable_output=human_readable,
                                  outputs=to_context,
                                  raw_response=result,
                                  output_prefix=?
                                  outputs_key_field=?
                                  )
```
discuss what should be the outputs prefix with @bakatzir if needed.

Reply:
Done